Program/Seasons.py

```diff
@@ -0,0 +1,222 @@
+import time
+
+
+
+#This is the stored pictures, calling "global" so the entire program can use them.
+setMediaPath("F:\COMP 120 Tinkering Graphics-JH182233\Pictures") 
+picfour = makePicture("Tree-Four-Seasons.jpg") 
+picwinter = makePicture("Tree-Winter.jpg") 
+picspring = makePicture("Tree-Spring.jpg") 
+picsummer = makePicture("Tree-Summer.jpg")
+picautumn = makePicture("Tree-Autumn.jpg")  
+picM = makeEmptyPicture(513, 556, black)
+pic = makeEmptyPicture(1025, 1110, black)
+
+
+#The main function for the program
+def main():
+    copy()
+    request()
+    request2()
+    
+    
+
+def copy():
+#create an empty picture and populate it with others 
+    startX = 0
+    startY = 0 
+    repaint (pic)
+    time.sleep(1)  
```

Comment:
What does 1 mean in this context? [Inline Comments](https://www.python.org/dev/peps/pep-0008/#inline-comments) are useful as clarifications in this context. Ensure that you indent inline comments to an appropriate level.